File: fixtures/typescript/renamed_function_duplicate/left.ts
export function buildOrderRows(orders: Order[]): Row[] {
  const paid = orders.filter((order) => order.paid);
  const local = paid.filter((order) => order.country === "US");
  const rows = local.map((order) => ({ id: order.id, total: order.amount }));
  const flagged = rows.filter((row) => row.total > 0);
  return flagged;
}

File: fixtures/haskell/renamed_function_duplicate/left.hs
module Orders where

buildOrderRows orders =
  let active = filter (\order -> order.amount > 0) orders
      rows = map (\order -> (order.id, order.amount, order.currency)) active
      total = foldl (\acc row -> acc + row.amount) 0 active
  in map
      (\row ->
        if row.amount > total
          then (row.id, row.currency, "large")
          else (row.id, row.currency, "standard"))
      active
